examples/wgpu_texture/src/demo_renderer.rs

```diff
@@ -172,7 +172,7 @@ impl ActiveDemoRenderer {
         // If "next texture" size doesn't match specified size then unregister and drop texture
         if let Some(next) = &self.next_texture {
             if next.texture.width() != width || next.texture.height() != height {
-                ctx.unregister_texture(next.handle);
+                ctx.unregister_texture(next.handle.clone());
                 self.next_texture = None;
             }
```

Comment:
Probably not a big deal, but this could become:

```rust
if self.next_texture.is_some_and(|tex| tex.width() != width || tex.height() != height) {
      let handle = self.next_texture().take().unwrap().handle;
      ctx.unregister_texture(handle);
}
```

to avoid the clone.